src/buf.rs

```diff
@@ -391,6 +391,32 @@ impl Buf {
         }
         tail
     }
+
+    /// Returns the element at the index, or `None` if the index is
```

Comment:
We should say `byte` instead of `element`. We don't operate anything except bytes.